fabtests/functional/rdm_stress.c

```diff
@@ -686,10 +907,11 @@ static int run_parent(void)
 		}
 	}
 
+	free_ctrls();
```

Comment:
There are a lot of `return` statements in the code blocks above this. Consider use `goto` to jump to here so the ctrls are freed (except for those from the child process).

Reply:
I tried to get away with this one and you caught me.  Will fix.